docs/fastdds/dds_layer/topic/contentFilteredTopic/createContentFilteredTopic.rst

```diff
@@ -26,9 +28,15 @@ Mandatory arguments are:
 Optional arguments are:
 
  * A string with the name of the filter class to use for the filter creation.
-   This allows the user to create filters different from the standard SQL like one.
+   This allows the user to create filters different from the standard SQL like one
+   (please refer to :ref:`dds_layer_topic_contentFilteredTopic_custom_filters`).
    Defaults to ``FASTDDS_SQLFILTER_NAME``.
 
+.. important::
+    An empty string as filter expression means that currently there is no filtering being done.
+    This could be of interest to enable/disable the DataReader filtering capabilities by simply
+    :ref:`updating the filter expression <dds_layer_topic_contentFilteredTopic_update>`.
```

Comment:
I'd rephrase this

```suggestion
    Setting an empty string as filter expression results in the disabling of the filtering (standard or otherwise).
    This can be used to enable/disable the filtering capabilities at any given time by simply
    :ref:`updating the filter expression <dds_layer_topic_contentFilteredTopic_update>`.
```

Reply:
I do not think we should mention the standard or custom filtering because it has not been introduced yet.